### ISSUE TITLE
[Feature-request] Make the suppress findings comment more descriptive

### ISSUE THREAD

**Issue by Matthew-Smith** [ajinabraham/njsscan#47]
It would be really nice if the inline comment to suppress findings included the tool's name similar to how [eslint works](https://eslint.org/docs/user-guide/configuring.html#disabling-rules-with-inline-comments):

```javascript
    res.redirect(target); // ignore: express_open_redirect
```

```javascript
    res.redirect(target); // njsscan-ignore: express_open_redirect
```

**Comment by ajinabraham** [comment]
Will consider this in the next release.

**Comment by ajinabraham** [comment]
Addressed in 0.2.2
